<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вход</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Для регистрации введите ФИО</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
